bitcoin/types_test.go

```diff
@@ -96,3 +57,12 @@ func Test_Block_UnmarshalJSON(t *testing.T) {
 		})
 	}
 }
+
+// loadTypeFixture takes a file name and returns the type fixture.
+func loadTypeFixture(fileName string) string {
+	content, err := ioutil.ReadFile(fmt.Sprintf("type_fixtures/%s", fileName))
+	if err != nil {
+		log.Fatal(err)
```

Comment:
nit: add some useful logging on failure? May be annoying goose-chase for some dev down the line if we are ever failing to load this fixture.

Reply:
Will do. I'll admit; I copied this one directly from https://github.com/rosetta-dogecoin/rosetta-dogecoin/blob/main/bitcoin/client_test.go#L1415 without much consideration beyond the directory change. 😄

Reply:
![Spider-Man-Pointing-Meme](https://user-images.githubusercontent.com/75836834/117526537-c0c5ee00-af7a-11eb-800b-56bd1d53c9ec.jpg)